docs/.vuepress/config.js

```diff
@@ -414,6 +414,54 @@ module.exports = {
         }
       ]
     },
+    '/ro/': {
+      lang: 'ro-RO',
+      label: translate('language'),
+      nav: [
```

Comment:
Looks like were missing the other nav items here. There's only 2, there should be 4.

Reply:
Totally missed it! Fixed in the last commit.
Thank you.